HEL1, HEL2, HEL3

mhelal opened the pull request:
training a tensor flow 5 layer RNN with 129 size, using stft magnitudes, and soft time masking

faroit commented:
Thanks for your submission. Please include a `description.md` in the root folder and move all json files into a subfolder like in the [submission-template](https://github.com/sigsep/sigsep-mus-2018/tree/master/submission-template)

mhelal commented:
Removed from the submissions folder, and added the tfss folder

mhelal commented:
Dear Fabian,

I added the file and moved the son files to folder tfss, but couldn't
rename the pull name from tfmss_RNN to tfss. I appreciate if you tell me
how. Searching google did not go well with me.

I also appreciate of you tell me the deadline for submitting another
version,

thanks,

Manal Helal,

On 2 April 2018 at 08:16, Fabian-Robert Stöter <notifications@github.com>
wrote:

> Thanks for your submission. Please include a description.md in the root
> folder and move all json files into a subfolder like in the
> submission-template
> <https://github.com/sigsep/sigsep-mus-2018/tree/master/submission-template>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sigsep/sigsep-mus-2018/pull/17#issuecomment-377872659>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABlx5tgUztBb0cJxgQykE36x4uyxrp10ks5tkdBSgaJpZM4TDAlj>
> .
>



-- 
Kind Regards,

Manal Helal

faroit commented:
Don‘t worry, I can rename the pull request. 

Do you mind if I rename you method to HEL1 later so that it is directly connected to your name.

When you add another method, just add another subfolder in your current pull request or create a new branch + pull request. 

Concerning the deadline its tomorrow 23:59 UTC

faroit commented on submissions/tfss/description.md:
```diff
@@ -0,0 +1 @@
+This is tensor flow 5 layer RNN trained neural networks using stft magnitudes and soft time masking. 
```
Please use the template provided by us and fill out the form so that we get some basic knowledge about you and the method

mhelal commented:
Dear Fabian,

I apologise for not reading all the details prior to submitting.

I created the HEL1 folder, with test subfolder, and followed the template
for the description.md file

I did not clean up the code yet, and did not upload it anywhere, and the
estimates wav file are 35mb each, and could not upload them now. My
colleague after the break might help me with some demo files.

Is this alright to pull now, without code, paper, or demo,


On 2 April 2018 at 12:14, Fabian-Robert Stöter <notifications@github.com>
wrote:

> *@faroit* requested changes on this pull request.
>
> As mentioned in our readme, the json files will have to go inside a
> subfolder called „test“. Please check again that you have exactly the same
> structure.
>
> Instead of deleting every file individually I would propose to make a new
> branch and create the folders again and then make a new pull request.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sigsep/sigsep-mus-2018/pull/17#pullrequestreview-108595123>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABlx5lDpP2auOfzx6sMkA1f9hs9-WECmks5tkggcgaJpZM4TDAlj>
> .
>



-- 
Kind Regards,

Manal Helal

faroit commented:
You now have 3 folders in your submission folder
you probably want to delete `tfss` and `tfss_Estimates` there

> I did not clean up the code yet, and did not upload it anywhere, and the estimates wav file are 35mb each, and could not upload them now. My colleague after the break might help me with some demo files.

Don't worry. This is all ok. Please __do dot__ commit/add any audio files to your PR branch. We will provide a way to upload the audio at a later stage. You don't need to provide any source code or paper.

faroit commented:
ok, great. this looks almost good now
Just remove this one readme and you‘re done: https://github.com/mhelal/sigsep-mus-2018/blob/cac576e87c6af7c83116a1cc3378517f84da17eb/submissions/HEL1/test/readme.md

faroit commented:
looks good to me now. Thanks for you submissions

faroit commented:
@mhelal After looking into the results I found that your SDR scores are almost all 0.0dB.  can you please check your audio results and verify that you want HEL2 to be submitted?

<img width="163" alt="default" src="https://user-images.githubusercontent.com/72940/38327829-8d7164d6-3849-11e8-8dd4-f089f2ba01b2.png">

mhelal commented:
The first 4 songs are producing the values below. I am printing these using
this command: print(scores)

and for each target such as:

print (scores.scores['targets'][2]['frames'][0]['metrics']['SDR'])


-0.561dB
-0.436dB
-10.539dB
-0.052dB
-0.038dB
0.065dB
-0.057dB
-0.044dB
-4.522dB
-0.000dB
-0.076dB
-0.000dB
0.051dB
-0.001dB
-0.000dB
0.025dB
Please advise if I am reading these values correctly or not, and if they
are worth submitting or not.

Sorry I am working on this project for a month and half only and was trying
to submit this year without much preparation,

Thanks,

On 4 April 2018 at 19:49, Fabian-Robert Stöter <notifications@github.com>
wrote:

> @mhelal <https://github.com/mhelal> After looking into the results I
> found that your SDR scores are almost all 0.0dB. can you please check your
> audio results and verify that you want HEL2 to be submitted?
>
> [image: default]
> <https://user-images.githubusercontent.com/72940/38327829-8d7164d6-3849-11e8-8dd4-f089f2ba01b2.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sigsep/sigsep-mus-2018/pull/17#issuecomment-378706031>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABlx5vcE94auq_RhHCEMQr80r5HBy8bZks5tlRXHgaJpZM4TDAlj>
> .
>



-- 
Kind Regards,

Manal Helal

faroit commented:
can you check if you maybe have a time shift in your estimates?

mhelal commented:
​
 bass.wav
<https://drive.google.com/file/d/1mOpPo6fg-RH2KYKd-Qyioz6geYK6wojD/view?usp=drive_web>
​This is an example file of AM Contra - Heart Peripheral (bass), by the ICA
method. I believe there is no time-shift, but not really sure if I can
measure it now,

Please advise if it is worth keeping it, or remove it until I debug
further,

thanks again,

On 5 April 2018 at 21:45, Fabian-Robert Stöter <notifications@github.com>
wrote:

> can you check if you maybe have a time shift in your estimates?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sigsep/sigsep-mus-2018/pull/17#issuecomment-379071115>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABlx5r4Nr06mqNprCu2hISfWXfXkqEP1ks5tloJ7gaJpZM4TDAlj>
> .
>



-- 
Kind Regards,

Manal Helal

faroit commented:
You are right: there is no time-shift. But the output of your estimate is almost 0, that is whats causing the close to 0dB SDR. 

I would suggest to remove HEL2 for now. But its of course up to you.

Cheers
Fabian

mhelal commented:
I just removed it,

Sorry for a difficult submission and thank you for your support,

Manal

On 5 April 2018 at 23:38, Fabian-Robert Stöter <notifications@github.com>
wrote:

> You are right: there is no time-shift. But the output of your estimate is
> almost 0, that is whats causing the close to 0dB SDR.
>
> I would suggest to remove HEL2 for now. But its of course up to you.
>
> Cheers
> Fabian
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/sigsep/sigsep-mus-2018/pull/17#issuecomment-379097638>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABlx5p6JbdNunYYSCBK9pRBEQ-O2znBGks5tlpz3gaJpZM4TDAlj>
> .
>



-- 
Kind Regards,

Manal Helal
